src/Spectre.libClassifier/UnsupportedDatasetTypeException.h

```diff
@@ -1,5 +1,5 @@
 /*
-* UnsupportedDatasetTypeException.h
+* UnsupportedDatasetTypeExceotion.h
```

Comment:
Typo. Exce**o**tion.

Reply:
I will fix it in next PR ok? I know maybe not the best strategy but to fasten the process of merging

Reply:
Or I think it is fixed in few later commits which aren't yet in allCommit commit ;] (DownSampledOpenCvDataset)

Reply:
If you see such errors, do not stage such lines to the commit please.